src/components/enterprise-user-subsidy/enterprise-offers/EnterpriseOffersBalanceAlert.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { useContext } from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+
+import { Container, Alert, MailtoLink } from '@edx/paragon';
+import { AppContext } from '@edx/frontend-platform/react';
+
+const EnterpriseOffersBalanceAlert = ({
+  adminText,
+  alertClassName,
+  alertVariant,
+  alertIcon,
+  alertHeading,
+  alertText,
+}) => {
+  const {
+    enterpriseConfig: { adminUsers },
+  } = useContext(AppContext);
+
+  const adminEmails = adminUsers.map(user => user.email);
+  const hasAdminEmails = adminEmails.length > 0;
+
+  const actions = hasAdminEmails ? [
+    <MailtoLink
+      to={adminEmails}
+      target="_blank"
+    >{adminText}
+    </MailtoLink>,
+  ] : [];
+
+  return (
+    <Container size="lg" className="pt-3">
+      <Alert
+        className={classNames({ [`${alertClassName}`]: hasAdminEmails })}
```

Comment:
[curious] Why would we only want to put on the class name conditionally when there are admin emails associated with the enterprise customer? If we're not targeting the class name anywhere, is it necessary?

Reply:
perhaps I misunderstood what was going on here, but previously there was a class name `'low-offers-balance-alert-with-cta'` here, so I imagined that if this alert was rendering the "no-balance" vs "low-balance" variant, that we would want a different class name here. class names are being determined in the utils file:

```
'no-offers-balance-alert-with-cta' : 'low-offers-balance-alert-with-cta',
```

Reply:
forget my previous comment. to more directly answer your question of `Why would we only want to put on the class name conditionally when there are admin emails associated with the enterprise customer?`: it was like that before, so I kept the logic. 🤷 i don't feel strongly about this though, so happy to remove if you think it's unnecessary